core.ts

```diff
@@ -204,6 +206,21 @@ export function PinejsClientCoreFactory(Promise: PinejsClientCoreFactory.Promise
 		}
 	}
 
+	// Escape a resource name (string), or resource path (array)
+	const escapeResource = (resource: string | string[]) => {
+		const encodedResource = baseEscapeResource(resource)
+
+		if (trailingCountRegex.test(encodedResource)) {
+			throw new Error('/$count used in an invalid place')
```

Comment:
Can we reverse the comment so that it instead suggests where count should be used? Eg “count can only be used on top level or expanded resources”.

Reply:
Sounds good